terraform/bastion_instance.tf

```diff
@@ -0,0 +1,41 @@
+resource "aws_instance" "bastion_instance_flugel_poc" {
+  ami           = "${lookup(var.AMIS, var.AWS_REGION)}"
```

Comment:
I see you are using data sources to get the latest image generated by Packer, but I do no see you are using the data source here to use that image. You should be using data source here.

Reply:
Added datasource in the bastion tf